project_input_template/README.md

```diff
@@ -3,3 +3,9 @@
 Copy this folder and move it out of the git repo before you use it. Make sure you set the `LANDBOSSE_INPUT_DIR` to point to the location of this copy of the folder. Alternatively, you can use the `--input` or `-i` command line options to set the path to this input folder.
 
 All the project files should have a name the matches the "Project ID" in `projects_list.xlsx` and should be placed in a `project_data` folder underneath the `projects_list.xlsx` file.
+
+## Versions of `project_list.xlsx`
+
+There are two version of `project_list.xlsx` in this folder. The first version is `project_list.xlsx`, which has all the inputs of LandBOSSE computed manually and is used for model validation. The second is `project_list_computed_inputs.xlsx`, which has formulas for the home run trench length and project construction duration, each as a function of number of turbines.
```

Comment:
I think the name of the second file is "project_list_simplified.xlsx" If so, that name should be changed here.

Reply:
Thanks for spotting the typo! I have corrected it.